lib/internal/vm/module.js

```diff
@@ -120,25 +118,29 @@ class Module {
       });
     }
 
+    let settings = {};
```

Comment:
I think we want to set null prototype otherwise we can trigger a user-defined `Object.prototype.vmReferer` setter
```suggestion
    let settings = { __proto__: null };
```